sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentPageContentService.java

```diff
@@ -83,7 +91,16 @@ public JsonArray getAnalysisMethodsAsJson(String experimentAccession) {
         return result;
     }
 
-    private JsonObject getExperimentFileAsJson(ExperimentFileType experimentFileType, String experimentAccession, String accessKey) {
+    public List<Publication> getPublications(List<String> identifiers) {
```

Comment:
I find it a bit weird that all methods in `ExperimentPageContentService` return `JsonElement`s but this one. See, in relation to this, my comment above in `ExperimentController.java`. I think the design could be improved to be more consistent.